source/includes/documentation/_vue_js_elements.md

```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
 
```

Comment:
1) It's good to have a phrase 'Parallax is located in the following class:'
2) Please, add this test also 'contentParallaxTests()'

Reply:
Modified as per the suggestions. Please review.